### PR TITLE
CLDR-14557 Modernize look up code or xpath (browse)

### PR DESCRIPTION
**Pull request by btangmu**
-New LookUp.vue partly working; revise layout; base locale only affects whatis

-Extend XPathAPI.java with api/xpath/str getByString to convert string to hex and dec id

-Move subroutine getXPathTable to bottom of XPathAPI.java

-Format XPathAPI.java with Eclipse

-New XPathRequest.java to enable POST request for api/xpath/str

-Change tt (obsolete html, triggered vue warning) to kbd

-Prevent addition of USE_DOJO to api/ requests

-Comments

##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-14557
- [x] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

File: tools/cldr-apps/src/main/webapp/js/esm/cldrLoad.js [hunk 1]
```diff
@@ -620,6 +620,7 @@ function getSpecial(str) {
   const specials = {
```
Comment (srl295) [inline tools/cldr-apps/src/main/webapp/js/esm/cldrLoad.js:620]: someday when all of these are vue, this lookup will be simpler!

**Review by srl295 — APPROVED** [review]
nice!

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/XPathRequest.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package org.unicode.cldr.web.api;
+
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+public class XPathRequest {
+
+    @Schema(required = true, description = "XPath string") public String str;
```
[the remark this answers is not in the document]
Comment (btangmu) [reply tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/XPathRequest.java:7]: good idea, i'll do a follow-up with that

**Comment by btangmu** [comment]
@srl295  I had to do resolve a little merge conflict, and so I need to ask for another review

**Review by srl295 — APPROVED** [review]
needs squash

**Review by srl295 — APPROVED** [review]

**Pull request merged**